website/docs/reference/resource-properties/unit-tests.md

```diff
@@ -155,7 +155,60 @@ When `format: csv`, can either supply:
 
 ### Overrides
 
-When configuring your unit test, you can override the output of macros, vars, or environment variables for a given unit test. 
+When configuring your unit test, you can override the output of [macros](https://docs.getdbt.com/docs/build/jinja-macros#macros), [project variables](https://docs.getdbt.com/docs/build/project-variables), or [environment variables](https://docs.getdbt.com/docs/build/environment-variables) for a given unit test. 
+
+```yml
+
+unit_tests:
+  - name: my_favorite_unit_test
+    model: my_favorite_model
+    overrides:
+      macros:
+        is_incremental: false 
+        current_timestamp: "date('2023-01-15')"
+      vars:
+        my_name: grace
+        platforms: ['web', 'mobile']
+      env_vars:
+        DBT_ENVIRONMENT_NAME: prod
+    ...
+
+```
+
+#### Macros
+
+You can override the output of any macro in your unit test defition. 
+
+There are some macros you _must_ override if the model your unit testing uses them:
+  - [`is_incremental`](https://docs.getdbt.com/docs/build/incremental-models#understanding-the-is_incremental-macro): If you're unit testing an incremental model, you must explicity set `is_incremental` to `true` or `false`. See more docs on unit testing incremental models [here](https://docs.getdbt.com/docs/build/unit-tests#unit-testing-incremental-models). 
+
+  ```yml
+
+  unit_tests:
+    - name: my_unit_test
+      model: my_incremental_model
+      overrides:
+        macros:
+          # unit test this model in "full refresh" mode
+          is_incremental: false 
+      ...
+
+  ```
+
+  - [`dbt_utils.star`](https://docs.getdbt.com/blog/star-sql-love-letter): If you're unit testing a model that uses the `star` macro, you must explicity set `star` to a list of columns. This is because the `star` only accepts a [relation](https://docs.getdbt.com/reference/dbt-classes#relation) for the `from` argument; the unit test mock input data is injected directly into the model SQL, replacing the `ref('')` or `source('')` function, causing the `star` macro to fail unless overidden.
```

Comment:
```suggestion
  - [`dbt_utils.star`](/blog/star-sql-love-letter): If you're unit testing a model that uses the `star` macro, you must explicity set `star` to a list of columns. This is because the `star` only accepts a [relation](/reference/dbt-classes#relation) for the `from` argument; the unit test mock input data is injected directly into the model SQL, replacing the `ref('')` or `source('')` function, causing the `star` macro to fail unless overidden.
```